projects/miopen/src/kernels/MIOpenIm2d2Col.cpp

```diff
@@ -56,7 +58,7 @@
 typedef char data_t;
 #elif MIOPEN_USE_INT32
 typedef int data_t;
-#elif(MIOPEN_USE_FP16 || MIOPEN_USE_BFP16)
+#elif (MIOPEN_USE_FP16 || MIOPEN_USE_BFP16)
```

Comment:
CI is complaining about this now

```suggestion
#elif(MIOPEN_USE_FP16 || MIOPEN_USE_BFP16)
```

Reply:
This is what clang-format did. Is the format file in the root of the rock-libraries repo correct?